### DISCUSSION TITLE
Setting Up Custom Arm

### DISCUSSION THREAD

**Discussion by qjgottschalk** [rainepretorius/olarm-ha-integration#13] [General]
Thank you for getting this up and running. 

Could you provide some details on how to bypass zones from HA and also if it's possible to setup a button on frontend that can arm the alarm and bypass predefined custom zones automatically?

**Comment by rainepretorius** [comment]
Hi,
This integration has no frontend except the default Home Assistant Alarm Panel. Thus, the frontend must be made self. To bypass a zone, check the doccumentation on the readme or the services.yaml for all the parameters.
